app/src/utils/validation/ValidSchema.tsx

```diff
@@ -0,0 +1,60 @@
+import * as Yup from 'yup';
+import { ReportCategories } from '@/models';
+
+// Validate names, emails, phone numbers, and addresses
```

Comment:
These look great!

For regex, can you leave a little comment on each of them to describe what it's checking for?

Otherwise, it's a lot of work for other developers to manually read through it :)